src/platform/monitoring/sentry.js

```diff
@@ -11,31 +11,26 @@ const trackErrors = environment.BASE_URL.indexOf('localhost') < 0;
 
 if (trackErrors) {
   const url = `${environment.BASE_URL}/js-report/0`.replace('//', '//faker@');
-  Raven.config(url, {
+  Sentry.init({
+    dsn: url,
     ignoreErrors: [
```

Comment:
So this one was tricky. It doesn't show up in the [documentation](https://docs.sentry.io/error-reporting/configuration/?platform=browser), but after searching through the [repo](https://github.com/getsentry/sentry-javascript/), I found the [`init` function](https://github.com/getsentry/sentry-javascript/blob/4d60369ee724365d8e24cdeda854cfa7c2f30b97/packages/browser/src/sdk.ts#L74) which takes a [`BrowserOptions`](https://github.com/getsentry/sentry-javascript/blob/4d60369ee724365d8e24cdeda854cfa7c2f30b97/packages/browser/src/backend.ts#L22) parameter, which extends [`Options`](https://github.com/getsentry/sentry-javascript/blob/master/packages/types/src/options.ts#L8), which has [`ignoreErrors`](https://github.com/getsentry/sentry-javascript/blob/master/packages/types/src/options.ts#L44), so this _should_ work the same as it does in `raven-js`. :crossed_fingers: